static/app/types/organization.tsx

```diff
@@ -132,6 +132,7 @@ export interface Member {
     'member-limit:restricted': boolean;
     'sso:invalid': boolean;
     'sso:linked': boolean;
+    'partnership:restricted'?: boolean;
```

Comment:
there are getsentry typescript and lint tests that fail if this is required -- will remove the `?` when the relevant files are updated